plugins/output-elasticsearch/src/main/scala/com/stratio/sparkta/plugin/output/elasticsearch/dao/ElasticSearchDAO.scala

```diff
@@ -22,11 +22,24 @@ import java.util.Date
 
 import org.joda.time.DateTime
 
+import com.stratio.sparkta.sdk.TypeOp
 import com.stratio.sparkta.sdk.TypeOp._
-import com.stratio.sparkta.sdk.{Output, TypeOp}
+import com.stratio.sparkta.sdk._
 
 trait ElasticSearchDAO extends Closeable {
 
+  final val TIMESTAMP_PATTERN = "@timestamp:"
+  final val DEFAULT_DATE_FORMAT = "YYYY.MM.dd"
+  final val YEAR = "YYYY"
```

Comment:
We need to update many contents of the documentation, we get into another PR. In addition to expanding the content to explaining all the new functionality.
If you notice there are very important things that they have changed drastically.